tensorflow_model_optimization/python/core/clustering/keras/clustering_registry.py

```diff
@@ -17,7 +17,7 @@
 import abc
 import six
 import tensorflow as tf
-
+from tensorflow.python.keras.engine.base_layer import TensorFlowOpLayer
```

Comment:
Please, use module-level imports, according to the [Google Python style guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#22-imports).

Reply:
Thanks,
I have updated it.